docs/content/SegmentedControl.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: SegmentedControl
+status: Draft
+description: Used to select an option from a short list and immediately apply the selection
+---
+
+## Examples
+
+### Simple
+
+```jsx live drafts
+<SegmentedControl aria-label="File view">
```

Comment:
How would you indicate which Button is selected?

Reply:
By passing `selected` to the `SegmentedControl.Button`

Reply:
If it'll be common for consumers to explicitly set the `selected` prop, could we show that in the examples? Similar to how `aria-current` is set in most of the `NavList` examples: https://primer-d954104e87-13348165.drafts.github.io/NavList